spark/v3.5/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveViews.scala

```diff
@@ -87,15 +86,6 @@ case class ResolveViews(spark: SparkSession) extends Rule[LogicalPlan] with Look
     }
   }
 
-  def loadView(catalog: CatalogPlugin, ident: Identifier): Option[View] = catalog match {
```

Comment:
some refactoring so that the same functionality can be used from different places